.jules/runs/bridge_bindings_wasm/pr_body.md

```
@@ -0,0 +1,60 @@
+## 💡 Summary
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)